File: Payroll Management/Part1/TestEmployee.java
import java.util.Scanner;
import java.util.ArrayList;
public class TestEmployee{
    public static void main(String args[]){
        ArrayList<Employee> employees = new ArrayList<Employee>();
        System.out.println("------------------------------------");
        while(true){
            System.out.println("DO YOU WANT TO ADD THE EMPLOYEE? yes or no");
            Scanner option = new Scanner(System.in);
            String choice = option.next();
            if(choice.equals("yes")){
                Scanner input = new Scanner(System.in);
                Employee employee = new Employee();
                System.out.print("ENTER THE EMPLOYEE DETAILS"+"\n"+"ENTER THE NAME :");
                String empname = input.nextLine();
                employee.setName(empname);
                System.out.println("ENTER THE EMPLOYEE DEPARTMENT :"+"\n"+"1. HUMAN RESOURCE"+"\n"+"2. IT"+"\n"+"3. FINANCE"+"\n"+"4. MARKETING"+"\n"+"5. R&D"+"\n"+"6. PRODUCTION");
                String empdepartment = input.nextLine();
                employee.setDepartment(empdepartment);
                 System.out.println("ENTER THE EMPLOYEE DESIGNATION"+"\n"+"1. TRAINEE ENGINEER"+"\n"+"2. SOFTWARE ENGINEER"+"\n"+"3. PROJECT LEAD"+"\n"+"4. PROJECT MANAGER"+"\n"+"5. PROGRAM MANAGER"+"\n"+"6. HR MANAGER");
                String empdesignation = input.nextLine();
                employee.setDesignation(empdesignation);
                System.out.println("ENTER THE EMPLOYEE SALARY :");
                String empsalary = input.nextLine();
                employee.setSalary(empsalary);
                employees.add(employee);
            }
            else{
                break;
            }
        }
        if(employees.size() == 0){
            System.out.println("!!No employee were added!!");
        }
        System.out.println("\n"+"EMPLOYEE DETAILS :");
        for(int i=0;i<employees.size();i++){
            System.out.println("\n");
            String emp_details = employees.get(i).toString();
            System.out.println(emp_details);
            employees.get(i).setAllowance();
            System.out.println("\n"+"\n"+"AFTER ALLOWANCE :");
            emp_details = employees.get(i).toString();
            System.out.println(emp_details);
        }
    }
}

File: Payroll Management/C_Sharp files/TestEmployee.cs
using System;
using System.Collections.Generic;
using EMP;
public class TestEmployee{
    public static void Main(string[] args){
        List<Employee> employees = new List<Employee>();
        while(true){
            Console.WriteLine("DO YOU WANT TO PROCEED? yes or no :");
            string choice = Console.ReadLine();
            if(choice.Equals("yes")){
                Employee employee = new Employee();
                Console.WriteLine("ENTER THE EMPLOYEE NAME WITH ID: "+employee.EmpId);
                string name = Console.ReadLine();
                employee.EmpName = name;
                Console.WriteLine("ENTER THE EMPLOYEE DEPARTMENT :"+"\n"+"1. HUMAN RESOURCE"+"\n"+"2. IT"+"\n"+"3. FINANCE"+"\n"+"4. MARKETING"+"\n"+"5. R&D"+"\n"+"6. PRODUCTION");
                string department = Console.ReadLine();
                employee.Department = department;
                Console.WriteLine("ENTER THE EMPLOYEE DESIGNATION"+"\n"+"1. TRAINEE ENGINEER"+"\n"+"2. SOFTWARE ENGINEER"+"\n"+"3. PROJECT LEAD"+"\n"+"4. PROJECT MANAGER"+"\n"+"5. PROGRAM MANAGER"+"\n"+"6. HR MANAGER");
                string designation = Console.ReadLine();
                employee.Designation = designation;
                Console.WriteLine("ENTER THE EMPLOYEE SALARY :");
                double salary = Double.Parse(Console.ReadLine());
                employee.Salary = salary;
                employees.Add(employee);
            }
            else{
                break;
            }
        }
        for(int i=0;i<employees.Count;i++){
            string details = employees[i].ToString();
            Console.WriteLine(details);
        }
    }
}
